db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_26_215928) do
+ActiveRecord::Schema.define(version: 2020_07_27_163200) do
```

Comment:
It looks that the schema is changing much more than comments, ad events and github repos though

Reply:
Shoot! good catch, too many migration branches in the air.